app/components/star/half_component.html.haml

```diff
@@ -1,6 +1,2 @@
 %svg.star.star--half{ width: "17", height: "14", viewBox: "0 0 17 15", fill: "none", xmlns: "http://www.w3.org/2000/svg" }
-  %path{ d: "M8.40002 0L10.1961 5.52786H16.0085L11.3062 8.94427L13.1023 14.4721L8.40002 11.0557L3.69774 14.4721L5.49385 8.94427L0.791572 5.52786H6.60391L8.40002 0Z", fill: "url(#paint0_linear_658_12679)" }
```

Comment:
Not sure if it's something in my local set up, but I can't see the half stars because of the display none: 
![image](https://github.com/citizensadvice/energy-comparison-table/assets/94383569/e99c9cad-5e2a-427e-9f1b-4d204a496a34)

Reply:
Ooh interesting! I've checked on a review app, and they seem to be showing up okay there. I'm not sure what in your set-up would stop them showing, though...

Reply:
I think this could be browser specific. I've now tried in Firefox and it works but it doesn't work for me in Chrome and Safari

Reply:
Oh no... me too! Will fix - thanks for spotting! Hmm, this change to Firefox might cause some problems...

Reply:
Now fixed!